.storybook/main.js

```diff
@@ -15,6 +15,10 @@ module.exports = {
         optimizationLevel: 2,
       },
     },
+    {
+      name: 'storybook-css-modules',
```

Comment:
Would we eventually move to https://www.npmjs.com/package/@storybook/addon-styling or is there something specific about `storybook-css-modules` that we're relying on 👀

Reply:
We should be able to move to the storybook addon once we upgrade to storybook v7